global_const.js

```diff
@@ -0,0 +1,11 @@
+// Property to determine if the importer should be run against the Pelias ES index that contains the polygon geo-shape data.
+global.geo_shape_polygon=true;
+
+// Validation for constants
+if(global.geo_shape_polygon === true || global.geo_shape_polygon === false){
```

Comment:
to check if the value is undefined you can use:
if (global.geo_shape_polygon !== undefined) 
more explanation here: https://stackoverflow.com/questions/858181/how-to-check-a-not-defined-variable-in-javascript

Reply:
Cheers Tomislav that's updated there now.